en/identity-server/7.1.0/docs/apis/restapis/idp.yaml

```diff
@@ -3280,4 +3409,338 @@ components:
         file:
           type: string
           format: binary
-          description: file to upload
\ No newline at end of file
+          description: file to upload
+  examples:
+    UserDefinedAuthenticatorPUTExample:
+      summary: "Put request for user defined authenticators"
+      value:
+        authenticatorId: "Y3VzdG9tLWF1dGhlbnRpY2F0b3I"
+        name: "custom-authenticator"
+        default: true
+        definedBy: "USER"
+        endpoint:
+          uri: "https://abc.com/token"
+          authentication:
+            type: "BASIC"
+            properties:
+              username: "auth_username"
+              password: "auth_password"
+    SystemDefinedAuthenticatorPUTExample:
+      summary: "Put request for system defined authenticators"
+      value:
+        authenticatorId: "U0FNTDJBdXRoZW50aWNhdG9y"
+        isEnabled:  true
+        isDefault:  true
+        properties:
+          - key: "somePropertyKey"
+            value: "somePropertyValue"
+    UserDefinedAuthenticatorsListPUTExample:
+      summary: "Put request for user defined authenticators list"
+      value:
+        defaultAuthenticatorId: "Y3VzdG9tLWF1dGhlbnRpY2F0b3I"
+        authenticators:
+          - authenticatorId: "Y3VzdG9tLWF1dGhlbnRpY2F0b3I"
+            name: "custom-authenticator"
+            default: true
+            definedBy: "USER"
+            endpoint:
+              uri: "https://abc.com/token"
+              authentication:
+                type: "BASIC"
+                properties:
+                  username: "auth_username"
+                  password: "auth_password"
+    SystemDefinedAuthenticatorsListPUTExample:
+      summary: "Put request for system defined authenticators list"
+      value:
+        defaultAuthenticatorId: "U0FNTDJBdXRoZW50aWNhdG9y"
+        authenticators:
+          - authenticatorId: "U0FNTDJBdXRoZW50aWNhdG9y"
+            isEnabled:  true
+            isDefault:  true
+            properties:
+              - key: "somePropertyKey"
+                value: "somePropertyValue"
+    IdentityProviderWithSystemDefinedAuthenticatorPOSTExample:
+      summary: "Request for identity provider with system defined authenticator"
```

Comment:
Can we say in summary as 'POST request for an identity provider with a system defined authenticator'